Source/Fuse.Common/NameValuePair.uno

```diff
@@ -8,10 +8,13 @@ namespace Fuse
 		Implements `IObject`, which means the `NameValuePair` can be viewed as an object
 		with a single property.
 
+		Implements `IArray`, which means the `NameValuePair` can be viewed as an array
+		with a single element.
+
 		An `IArray` containing some `NameValuePairs` can be converted to an `IObject`
 		implementation containing all those properties using the `ObjectFromArray` method.
 	*/
-	public sealed class NameValuePair : IObject
+	public sealed class NameValuePair : IObject, IArray
```

Comment:
Isn't this supposed to be something handled by marshalling? Couldn't any individual object, which isn't already an array, be marshallable to an array?

This isn't special with NVP here, a normal integer, string, or otherwise should be assignable to an array. There is another issue about that: https://github.com/fusetools/fuselibs-public/issues/253